data/src/config/buffer.rs

```diff
@@ -73,7 +77,36 @@ impl Default for ServerMessage {
             enabled: true,
             smart: Default::default(),
             username_format: UsernameFormat::default(),
+            exclude: Default::default(),
+            include: Default::default(),
+        }
+    }
+}
+
+impl ServerMessage {
+    pub fn should_send_message(&self, channel: Option<&str>) -> bool {
```

Comment:
Nit: Remove Option here and change how it's handled at call site. The option has no need within this funciton